tools/CMakeLists.txt

```diff
@@ -4,3 +4,12 @@ endif()
 if (BUILD_NOISE_TOOLS)
     add_subdirectory(noise)
 endif()
+
+install(
+  FILES
+      common.sh  purge_from_cache.sh  purge_non_existing_images.sh  purge_unused_tags.sh
```

Comment:
should we also consider dtstyle_to_xmp.py ?

Reply:
which then begs the question if we should move all the "non end user tools" into a subdirectory (or vice versa) to make it easier to know what we should install with "make install"

From a packager point of view ... the python script adds python-elementtree as dependency. also before we start shipping it ... we should make it is python3 ready. many of the big distros are moving to python3 only as python2 is EOL end of the year.

Reply:
so I would not put that into this PR but into a 2nd and check if we are py3 ready.

Reply:
Ok, makes sense!